<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntegralEquationClass</class>
 <widget class="QMainWindow" name="IntegralEquationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вычисление интеграла методом Келлога</string>
  </property>
  <property name="windowIcon">
   <iconset resource="IntegralEquation.qrc">
    <normaloff>:/IntegralEquation/Resources/integral-mathematical-sign.png</normaloff>:/IntegralEquation/Resources/integral-mathematical-sign.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QtCharts::QChartView" name="grafikWidgetZkr" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QtCharts::QChartView" name="grafikWidgetZ2" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableWidget" name="twMu1">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>71</height>
         </size>
        </property>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="twMu2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>71</height>
         </size>
        </property>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Данные</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QComboBox" name="cbCore">
       <item>
        <property name="text">
         <string>sqrt(x^2+t^2)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ln(1+xt)</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>a</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>b</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Количество разбиений</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Количество итераций</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Точность</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSpinBox" name="sbA"/>
         </item>
         <item>
          <widget class="QSpinBox" name="sbB">
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbKolRazbieniy">
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="value">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="sbKolIterac">
           <property name="value">
            <number>11</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="dsbTochnost">
           <property name="decimals">
            <number>6</number>
           </property>
           <property name="value">
            <double>0.000100000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblSobstvennoeChislo">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>13</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 255);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblSobstvennoeChislo_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>13</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(23, 156, 8);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="IntegralEquation.qrc"/>
 </resources>
 <connections/>
</ui>
